tests/retention-data/retention.ts

```diff
@@ -77,11 +82,113 @@ test('it should calculate the rigth retention data for 28 days', (t) => {
  * 4A 1011
  * 4B 1100
  */
-test('it should calculate the rigth retention data for 4 days', (t) => {
+test('it should calculate the right retention data for 4 days', (t) => {
     const now = new Date();
     const activities = parseActivities(activities4daysRaw);
     const result = getRetentionData(activities, now);
 
     t.is(result.oneDay, 5);
     t.is(result.twoDays, 4);
 });
+
+/*
+ * 1A 1111 X
+ * 1C 1111 X
+ * 1F 1111 X
+ * 2A 1111 X
+ * 2C 1111 X
+ * 2D 1111
+ * 3B 1111 X
```

Comment:
The comment is a bit misleading since there's no data for `B` in day `1` or `2`. The value of `1111` on day `3` suggests that there should be data on those days. Furthermore you list `5` unique users (`A`, `B`, `C`, `D`, and `F`) in the comment, but only show `3` unique users were active in the test.